NanaZip.ExtensionPackage/NanaZip.ExtensionPackage.Installer.iss

```diff
@@ -0,0 +1,32 @@
+﻿#define AppName "NanaZip Extension Package"
+#define AppPublisher "M2-Team"
+#define AppCopyright "Copyright (c) M2-Team and Contributors. All rights reserved."
+#define AppURL "https://github.com/M2Team/NanaZip"
+
+#ifndef AppVersion
+#define AppVersion "5.1.0.0"
+#endif
+
+[Setup]
+AppId={{42795434-AB1A-4197-A724-F13E08953DFC}
```

Comment:
It seems the AppId's GUID format has some issues.

Reply:
This is due to a quirk of Inno Setup's bracket quoting rules (only the open bracket is quoted).